Source/santad/EventProviders/SNTEndpointSecurityFileAccessAuthorizer.mm

```diff
@@ -635,10 +671,33 @@ - (FileAccessPolicyDecision)handleMessage:(const Message &)msg
       event.pid = @(audit_token_to_pid(msg->process->audit_token));
       event.ppid = @(audit_token_to_pid(msg->process->parent_audit_token));
       event.parentName = StringToNSString(msg.ParentProcessName());
+      event.signingChain = cd.certChain;
+
+      if (!policy->silent && self.fileAccessBlockCallback) {
+        self.fileAccessBlockCallback(event, OptionalStringToNSString(policy->custom_message));
+      }
 
-      self.fileAccessBlockCallback(event);
+      if (ShouldMessageTTY(policy, msg, self->_ttyMessageCache)) {
+        NSAttributedString *attrStr =
+          [SNTBlockMessage attributedBlockMessageForFileAccessEvent:event
+                                                      customMessage:OptionalStringToNSString(
+                                                                      policy->custom_message)];
+
+        NSMutableString *blockMsg = [NSMutableString stringWithCapacity:1024];
+        [blockMsg appendFormat:@"\n\033[1mSanta\033[0m\n\n%@\n\n", attrStr.string];
```

Comment:
nit: can `\033[1m` be brought out to a constant describing what it does?

Reply:
Discussed internally. Due to some various weirdness and limited use cases, not going to spend time on a more general solution. Commented the escape sequences for now.